### ISSUE TITLE
Randomly not working at all

### ISSUE THREAD

**Issue by OlazRed** [DestinyItemManager/DIM#2621]
While using, DIM doesn't work from time to time, for a couple of hours, then works again. An error appears, when this happens (attached).

This has begun around the last patch in D2 (a week ago). Checked on a couple of machines in different places.

![dim_error](https://user-images.githubusercontent.com/36138817/35782345-ce851232-09f6-11e8-96eb-a2e738d22f67.jpg)

**Comment by bhollis** [comment]
When this happens, can you wait a couple seconds then hit the refresh button and have it work? If not, as the error states, you may have gotten updated rules for an extension like AdBlock or Privacy Badger that are blocking bungie.net.

If you aren't able to get it working, please let us know what browser you're seeing this in.

**Comment by OlazRed** [comment]
Refreshing the website doesn't help. I always try to do that, when the bug happens, and after a couple of tries I give up and check it an hour or so later. This happens on disabled AdBlock and firewall. Tested on: Chrome, Firefox, Opera and Edge. When the bug occurs, it does on all the browsers at the same time. Can't say it happens for any specific reason, at least not one I've been able to identify. I'm using DIM almost constantly when playing, and I can say that it happens pretty infrequently. A couple of colleagues also experience this bug at the same time as I (not all of them though).

**Comment by duckmanBR** [comment]
Next time this happens, can you try to access DIM using your phone on mobile data (not wifi)?

**Comment by OlazRed** [comment]
Sure. I'll let you know the results when I do.

**Comment by OlazRed** [comment]
Hey, I've been able to test it just now on mobile data, and it works perfectly fine, while not working on wifi. Taking this into the consideration, I believe it is some internet provider issue?

BTW, I've just noticed, that when the "bug" occurs, an error 522 appears on Bungie's website. I believe this may be connected to each other.

![fx9r71u](https://user-images.githubusercontent.com/36138817/35873894-6690360e-0b6b-11e8-9be7-1963446a9507.png)
